.github/workflows/master_dev_checks.yml

```diff
@@ -0,0 +1,409 @@
+---
+  name: Best Practices Check
+  on:
+    workflow_call:
+    
+  jobs:
+
+    dev-checks:
+      runs-on: non-build
+      steps:
+        - name: Dump GitHub context
+          env:
+            GITHUB_CONTEXT: ${{ toJson(github) }}
+          run: |
+            echo "$GITHUB_CONTEXT"
+
+        - uses: actions/checkout@v3
+          with:
+            token: ${{ secrets.CIPIPELINE_NEXUS_PASSWORD  }}
+            fetch-depth: 0
+
+        - name: Get branch name (merge)
+          if: github.event_name != 'pull_request'
```

Comment:
When will this be the case? I thought you told all workflows are on `pull_request` event?